.github/workflows/main.yml

```diff
@@ -66,9 +66,9 @@ jobs:
           python -m poetry run bash scripts/lint.sh
 
       - name: Test
-        run: python -m poetry run bash scripts/test-coverage-xml.sh unit
+        run: python -m poetry run bash scripts/test-coverage-xml.sh
```

Comment:
Why did we get rid of the `unit` here? I thought we wanted integration tests in main?

Reply:
Removing unit means that now both unit and integration tests are happening

Reply:
@htahir1 from what @AlexejPenner explained to me, when you specify `unit` it only runs the unit tests, whereas if you don't specify the argument it will run all tests (unit & integration at the moment)

Reply:
Ah you're right! Ok ill resolve the comment!